blog/templates/blog/delete_entry.html

```diff
@@ -0,0 +1,5 @@
+{% extends "blog/blogbase.html" %}
+
+{% block blog_content %}
+    <h2>Are you sure you want to delete {{ entry.title }}?</h2>
```

Comment:
&lt;h2> are header tags.  When asking a question like this, you'd want to use a &lt;p> (paragraph) tag since it's just text to a user (not displayed as a header).